molecule/__init__.py

```diff
@@ -18,8 +18,19 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
+import signal
+import sys
+
 from pbr import version
 
+
+def signal_handler(signal, frame):
+    print('\033[31mExiting molecule on Ctrl+c.\033[0m')
```

Comment:
Prints the message to the console using ANSI red without having outside dependencies on anything else (like colorama).